examples/azure-functions-deployment/az_deploy.out

```diff
@@ -1,26 +1,12 @@
-$ az functionapp deployment source config-zip -g functions1777 -n functions1777 --src  <project_dir>/target/bin/azure-functions.zip
-Getting the SCM site credentials for the ZIP file deployment.
-Starting the ZIP deployment. This operation can take a while to complete ...
-The deployment endpoint responded with the status code 202.
-{
-"active": true,
-"author": "N/A",
-"author_email": "N/A",
-"complete": true,
-"deployer": "ZipDeploy",
-"end_time": "2020-07-02T06:48:08.7706207Z",
-"id": "2bacf185fb114d42aab762dfd5f303dc",
-"is_readonly": true,
-"is_temp": false,
-"last_success_end_time": "2020-07-02T06:48:08.7706207Z",
-"log_url": "https://functions1777.scm.azurewebsites.net/api/deployments/latest/log",
-"message": "Created via a push deployment",
-"progress": "",
-"provisioningState": null,
-"received_time": "2020-07-02T06:47:56.2756472Z",
-"site_name": "functions1777",
-"start_time": "2020-07-02T06:47:56.7600364Z",
-"status": 4,
-"status_text": "",
-"url": "https://functions1777.scm.azurewebsites.net/api/deployments/latest"
-}
+$ func azure functionapp publish bal-bbe --script-root target/azure_functions
```

Comment:
```suggestion
$ func azure functionapp publish bal-bbe --script-root target/azure_functions
```
These commands should go to the `.md` file. Only the CLI output should be there in the `.out` file.